File: C#/Beecrowd/1036 - Bhaskara's Formula.cs
using System;

namespace Beecrowd{
    
    class URI {

       static void Main(string[] args) { 
            
          string[] scanf = Console.ReadLine().Split();
          double a = double.Parse(scanf[0])/10;
          double b = double.Parse(scanf[1])/10;
          double c = double.Parse(scanf[2])/10;
          double delta = b*b - 4*a*c;
          double x1 = (b*(-1) + Math.Sqrt(delta))/(2*a);
          double x2 = (b*(-1) - Math.Sqrt(delta))/(2*a);

          if(a>0 && delta>=0){
            Console.WriteLine($"R1 = {x1:0.00000}");
            Console.WriteLine($"R2 = {x2:0.00000}");
          }
          else{
            Console.WriteLine("Impossivel calcular");
          }
       }
    }
}

File: JavaScript/Beecrowd/1036 - Bhaskara's Formula.js
// Reading the input values
let input = require('fs').readFileSync('/dev/stdin', 'utf8');
let lines = input.split('\n');

let nums = input.split(" ");
let A = parseFloat(nums[0]);
let B = parseFloat(nums[1]);
let C = parseFloat(nums[2]);

// Calculating the discriminant
let delta = Math.pow(B, 2) - 4 * A * C;

// Checking if it's possible to calculate the roots
if (A === 0 || delta < 0) {
  console.log("Impossivel calcular");
} else {
  // Calculating the roots
  let R1 = (-B + Math.sqrt(delta)) / (2 * A);
  let R2 = (-B - Math.sqrt(delta)) / (2 * A);

  // Printing the roots
  console.log("R1 = " + R1.toFixed(5));
  console.log("R2 = " + R2.toFixed(5));
}
